### DISCUSSION TITLE
[feature]: Support other databases

### DISCUSSION THREAD

**Discussion by Heartnett** [undb-xyz/undb#1356] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Please add support for MySql, MariaDB, PostgreSQL, and other databases. 

### Why should this be worked on?

Other databases should be supported for platform flexibility.

**Comment by ermaxinc** [comment]
MySql +++

**Comment by ghost** [comment]
> Please add support for MySql, MariaDB, PostgreSQL, and other databases.

The most important feature would be the "browser local storage" (IndexedDB, WebSQL, localStorage, cookies, sessions, sqlite3 wasm, PGlite(postgresql wasm) or couch/pouchdb) and "browser external offline storage" (MySQL, MariaDB, PostgreSQL, SQL Server, SQLite, etc). Maybe this would be very easy to set up and can work on any machine with little processing power or on older machines.  Or it would be interesting to have a Raspberry Pi with this.
